dts/bindings/flash_controller/sim-flash.yaml

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+title: simulated flash
+version: 0.1
+
+description: >
+    This binding gives a base representation of a simulated flash memory
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "sim-flash"
```

Comment:
This should be "zephyr,sim-flash"

Reply:
I will also rename the file as it seems that this constraint value is aligned with the yaml file name in other examples